test/algconvergence/ode_convergence_tests.jl

```diff
@@ -37,8 +38,8 @@ testTol = 0.2
   sim3 = test_convergence(dts2,prob,RKO65())
   @test sim3.𝒪est[:l∞] ≈ 5 atol=testTol
 
-  sim3 = test_convergence(dts4,prob,FRK65(0)) #standard value for non zero-dissipation tests
-  @test sim3.𝒪est[:l∞] ≈ 6 atol=testTol
+  sim3 = test_convergence(dts4,prob,FRK65())
+  @test sim3.𝒪est[:l∞] ≈ 6 atol=testTol1
```

Comment:
instead, make this 6.5 and keep the same testtol. Looks like super convergence. What does plot(sim3) look like?

Reply:
Here's the plot: 
![plot](https://user-images.githubusercontent.com/37050056/75071212-ec49ed80-551a-11ea-8134-542dfd8cbc93.png)
`sim3.�est[:l∞]=6.547286289246398`
![plot1](https://user-images.githubusercontent.com/37050056/75071214-ece28400-551a-11ea-91a1-c67f347dce1c.png)
`sim3.�est[:l∞]=5.862512678209683`

Reply:
huh, that's weird. Instead of defining at `testTol1`, just put the larger value right in the code there and I'll accept.

Reply:
Done. By the way, what is super convergence?